vue.config.js

```diff
@@ -31,5 +32,15 @@ module.exports = vueUtils({
         toType: 'dir',
         ignore: ['.DS_Store'],
       }]]);
+
+    // limit highlight.js to only the necessary languages
+    const builtinLanguages = 'bash|c|s?css|cpp|diff|http|java|llvm|perl|php|python|ruby|xml|javascript|json|markdown|objectivec|shell|swift';
+    const envLanguages = (process.env.VUE_APP_HLJS_LANGUAGES ?? '').split(',').join('|');
+    config
+      .plugin('languages')
+      .use(webpack.ContextReplacementPlugin, [
+        /highlight\.js\/lib\/languages$/,
+        new RegExp(`\/(${[builtinLanguages, envLanguages].join('|')})$`),
```

Comment:
I think you dont need to escape here, since you are writing a string inside a RegExp, it will escape it for you.

```suggestion
        new RegExp(`/(${[builtinLanguages, envLanguages].join('|')})$`),
```